allennlp/data/fields/list_field.py

```diff
@@ -24,7 +25,7 @@ class ListField(SequenceField):
         A list of ``Field`` objects to be concatenated into a single input tensor.  All of the
         contained ``Field`` objects must be of the same type.
     """
-    def __init__(self, field_list: List[Field]) -> None:
+    def __init__(self, field_list: List[Field]) -> None:  # pylint: disable=super-init-not-called
```

Comment:
You're right that fixed it .... The way these types interact with pylint seems a bit magical to me.